### PR TITLE
3 State Alignment

### PR DESCRIPTION
**Pull request by bioinsilico**
- 3 color code alignment
- Custom tooltip
- All deps updated

### CODE REVIEW

**Comment by bioinsilico** [comment]
Btw built with node 21

File: src/utils/sequence-tooltip.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { RcsbFvTooltipInterface } from '@rcsb/rcsb-saguaro/lib/RcsbFv/RcsbFvTooltip/RcsbFvTooltipInterface';
+import {
+    RcsbFvTrackDataAnnotationInterface
+} from '@rcsb/rcsb-saguaro-app/lib/RcsbFvWeb/RcsbFvFactories/RcsbFvTrackFactory/RcsbFvTrackDataAnnotationInterface';
+
+export class SequenceTooltip implements RcsbFvTooltipInterface {
+
+    private readonly STRUCTURAL_ALIGNMENT = 'STRUCTURAL ALIGNMENT';
+    private readonly SEQUENCE_ALIGNMENT = 'SEQUENCE ALIGNMENT';
+    private readonly UNALIGNED_REGION = 'UNALIGNED REGION';
```
Comment (valasatava) [inline src/utils/sequence-tooltip.ts]: Perhaps, "alignment unavailable"?

**Review by valasatava — COMMENTED** [review]
Thank you, @bioinsilico for implementation and taking care of dependencies update! Just one question with respect to naming convention

File: src/utils/sequence-tooltip.ts (hunk 1 above)
Comment (bioinsilico) [reply src/utils/sequence-tooltip.ts]: Changed :)

**Review by valasatava — APPROVED** [review]
Thank you very much @bioinsilico! Please, feel free to release to prod

**Pull request merged**